wormchain/go.mod

```diff
@@ -136,7 +136,8 @@ require (
 
 replace (
 	github.com/99designs/keyring => github.com/cosmos/keyring v1.1.7-0.20210622111912-ef00f8ac3d76
-	github.com/CosmWasm/wasmd v0.30.0 => github.com/wormhole-foundation/wasmd v0.30.0-wormchain-1
+	// github.com/CosmWasm/wasmd v0.30.0 => github.com/wormhole-foundation/wasmd v0.30.0-wormchain-1
+	github.com/CosmWasm/wasmd => github.com/wormhole-foundation/wasmd v0.30.0-wormchain-1.0.20230706174913-e99aa2ac4302
```

Comment:
TEMP CHANGE until dependent PR https://github.com/wormhole-foundation/wasmd/pull/13 is merged!